packages/core/src/IdbAdapter.ts

```diff
@@ -66,9 +66,9 @@ export class IdbAdapter extends StorageAdapter {
     await this.database!.add('changes', changeSet)
   }
 
-  async putSnapshot(documentId: string, snapshot: Snapshot) {
+  async putSnapshot(documentId: string, snapshot: Snapshot, clock: Clock) {
     this.ensureOpen()
-    await this.database!.put('snapshots', { documentId, snapshot })
+    await this.database!.put('snapshots', { documentId, snapshot, clock })
   }
 
   async deleteSnapshot(documentId: string) {
```

Comment:
Clocks and Snapshots are now stored side-by-side, so we need to implement that both here and in the abstract StorageAdapter class.